src/AddTodoForm.js

```diff
@@ -10,32 +11,25 @@ function AddTodoForm({ onAddTodo }) {
             id: Date.now()
         };
         onAddTodo(newTodo);
-        console.log(todoTitle);
         setTodoTitle('');
     }
-    
-
-    function handleTitleChange(event) {
-        const newTodoTitle = event.target.value;
-        setTodoTitle(event.target.value);
-    }
-
 
     return (
         <form onSubmit={handleAddTodo}>
-            <label htmlFor="todoTitle">Title</label>
-            <input 
-                name="title" 
-                type="text" 
-                id="todoTitle" 
-                placeholder="Enter title here..." 
+            <InputWithLabel
+                name="title"
+                type="text"
+                id="todoTitle"
+                placeholder="Enter title here..."
```

Comment:
I'm a little torn on this because the lesson is about creating reusable components, and that's not really what the content of the lesson demonstrated. What you've done is great and is much more reusable but I'll paste in what the directions were asking for. No need to change anything, great work!

```
      <InputWithLabel
        todoTitle={todoTitle}
        handleTitleChange={handleTitleChange}
      >
        Title
      </InputWithLabel>

```